vnext/Desktop.ABITests/JsiTests.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include <gtest/gtest.h>
+
+// JSI
+#include <jsi/test/testlib.h>
+
+// NAPI
+#include <Shared/JSI/NapiJsiRuntime.h>
+
+using facebook::jsi::Runtime;
+using facebook::jsi::RuntimeFactory;
+
+namespace Microsoft::Test {
+
+std::vector<RuntimeFactory> runtimeGenerators() {
+  return {
+#ifdef USE_V8
+      []() -> std::unique_ptr<Runtime> {
+        napi_ext_env_settings settings{};
+
+        settings.this_size = sizeof(settings);
+        settings.flags.enable_gc_api = true;
+        napi_env env{};
+        napi_ext_create_env(&settings, &env);
+
+        return Microsoft::JSI::MakeNapiJsiRuntime2(env);
```

Comment:
>MakeNapiJsiRuntime2 [](http://example.com/codeflow?start=31&length=19)

This is non-ABI safe API. The tests are suppose to test only the ABI-safe APIs.

Reply:
This non-ABI safe API is both defined and used within the RNW DLL.
Is this a problem at all?

Reply:
Thanks for the off-line clarification.

This methods is not meant to be exported, and for non-RNW projects consuming it, we will export the source files as part of the NuGet package.